account_bank_statement_import_online/__init__.py

```diff
@@ -0,0 +1,5 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import models
+from . import wizards
+from .tests import online_bank_statement_provider_dummy
```

Comment:
Why this?

Reply:
Otherwise that model won't be available during tests. It's either this (and placing it under tests) or moving it to models